PROJECT

```diff
@@ -0,0 +1,15 @@
+domain: openshift.io
+layout: go.kubebuilder.io/v2
+repo: github.com/openshift/elasticsearch-operator
+resources:
+- group: logging
+  kind: Elasticsearch
+  version: v1
+- group: logging
+  kind: Kibana
+  version: v1
+multigroup: true
+version: 3-alpha
+plugins:
+  go.operator-sdk.io/v2-alpha: {}
```

Comment:
This plugin name changed in operator-sdk to `go.sdk.operatorframework.io/v2-alpha` in operator-sdk 1.0.0: 
https://v1-1-x.sdk.operatorframework.io/docs/upgrading-sdk-version/v1.0.0/#upgrade-your-project-from-version-2-to-3-alpha